dll/win32/shell32/CQueryAssociations.cpp

```diff
@@ -641,13 +649,15 @@ HRESULT CQueryAssociations::GetCommand(const WCHAR *extra, WCHAR **command)
     RegCloseKey(hkeyShell);
     if (ret)
     {
+        ERR("0x%lX\n", ret);
```

Comment:
Not sure we need more pollution in the debug log.

Reply:
```suggestion
```